Tools/build.bat

```diff
@@ -189,6 +190,34 @@ set __VERBOSITY__=n
 msbuild.exe %BUILD_FILE% /t:%__TARGET__% /p:Configuration="%~1" /P:Platform=%2 -fileLoggerParameters:Verbosity=%__VERBOSITY__%;LogFile=%BUILD_LOG_FILE%
 goto :eof
 
+:run_semmle
+if "%SEMMLE_HOME%"=="" (
+  echo "Semmle path is not configured, skipping"
+  goto :eof
+)
+
+echo "Removing previously created rules database"
+rmdir /s/q semmle_db
+
+echo "Creating rules database for %BUILD_FILE%"
+echo msbuild.exe %BUILD_FILE% /t:rebuild /p:Configuration="%~1" /P:Platform=%2 > semmle_bld.bat
+call %SEMMLE_HOME%\codeql\codeql.exe database create -l=cpp -c "semmle_bld.bat" semmle_db -j 0
+call del semmle_bld.bat
+
+IF ERRORLEVEL 1 (
+  set BUILD_FAILED=1
+  goto :eof
+)
+
+echo "Analyzing rules database for %BUILD_FILE%"
+call %SEMMLE_HOME%\codeql\codeql.exe database analyze semmle_db "%SEMMLE_HOME%\Windows-Driver-Developer-Supplemental-Tools\codeql\windows-drivers\suites\windows_driver_mustfix.qls" --format=sarifv2.1.0 --output="%BUILD_NAME%.sarif" -j 0 --rerun
```

Comment:
I preferred to use all suites, not only windows_driver_mustfix. DVL ignores errors that are not **mustfix**, but we can look at it.

Reply:
Agree